### PR TITLE
Luca Dillenburg / Add my contribution 

### PR DESCRIPTION
**Pull request by LucaDillenburg**
Unfortunately, I am leaving Dextra at the end of the month :sob: :sob: , but I am submitting it because I am a Dextra employee today. However, I completely understand if it doesn't get accepted. :smile:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ Dextra is a Software development company which is really proud of the great work
  - [JavaScript](#javascript-)
  - [.NET](#net-)
  - [TypeScript](#typescript-)
+ - [GoLang](#golang-)
```
Comment (erickzanardo) [inline README.md:32]: This list mus follow an alphabetical order

File: README.md [hunk 2]
```diff
@@ -42,6 +43,7 @@ Dextra is a Software development company which is really proud of the great work
 
  - [Nanoflux](https://github.com/ohager/nanoflux) - Very lightweight, depency-free Flux implementation. Author: [Oliver Hager](https://github.com/ohager)
  - [Stappo](https://github.com/ohager/stappo) - Extremely tiny Application State Manager written in Javascript (less than 300 bytes). Author: [Oliver Hager](https://github.com/ohager)
+ - [Engine-2D-Shooting-Game](https://github.com/lucadillenburg/Engine-2D-Jogo-de-Tiro/) - A top-down 2D shooting game engine. Author: [Luca Dillenburg](github.com/LucaDillenburg)
```
Comment (erickzanardo) [inline README.md:46]: This list must follow an alphabetical order

File: README.md [hunk 3]
```diff
@@ -51,6 +53,10 @@ Dextra is a Software development company which is really proud of the great work
 <h1>.NET <img style="margin: 4px 0 0 4px" height="32" src="media/dotnet.svg" alt=".NET Logo"/></h1>
 
  - [DexQuiz](https://github.com/dextra/dexquiz) - A simple quiz with different tracks, to be used in events to engage people with the company and also help with gifts and awards. Author: [Gustavo Bigardi](https://github.com/gustavobigardi)
+ 
+<h1>GoLang <img style="margin: 4px 0 0 4px" height="32" src="media/golang.png" alt="GoLang Logo"/></h1>
```
Comment (erickzanardo) [inline README.md:57]: This list must follow an alphabetical order

**Review by erickzanardo — CHANGES_REQUESTED** [review]
Just need to fix the alphabetical order of the items, otherwise, LGTM

File: README.md [hunk 4]
```diff
@@ -51,6 +53,10 @@ Dextra is a Software development company which is really proud of the great work
...
+
+ - [Open-Computer](https://github.com/Open-Machine/) - A computer built from scratch: from the circuit to the assembler (high-end language coming soon). Author: [Luca Dillenburg](github.com/LucaDillenburg)
```
Comment (ohager) [inline README.md:59]: I looked into the project. The idea is nice, but it's in a very raw state, such I cannot see the representational value for Dextra. Good luck with your project.

File: README.md (hunk 2 above)
Comment (ohager) [inline README.md:46]: The presentation is good and looks promising. But the projects structure and the code lacks representational quality. Unfortunately, I do not recommend to put this onto the awesome-list. Don't get me wrong: continue studying and doing things you're interested in. That's the way we all evolve! Heads up

**Review by ohager — COMMENTED** [review]
Interesting projects, but one lacks completeness and the other quality - to sum it up.

File: README.md (hunk 4 above)
Comment (LucaDillenburg) [reply README.md:59]: Yes, I agree. Thank you for your feedback! I will be working in it soon!

File: README.md (hunk 2 above)
Comment (LucaDillenburg) [reply README.md:46]: No problem! I developed this project before studying unit testing and other clean code concepts! Thank you for your feedback anyway!

File: README.md (hunk 4 above)
Comment (ohager) [reply README.md:59]: We discussed the Open Machine stuff...as I said before: pretty cool idea, and we all liked it. If you could evolve more on that, such there is kind of working example, we might pull it into the list. 
I read through your README and saw that certain commands are not there (`assembler`), broken link (to circuit folder) and so on. I'm not sure, if you have something functional - if so it seems not to be commited.

**Comment by ohager** [comment]
@erickzanardo @gustavobigardi I intent to close this PR.  We could pull it to the upcoming "Promising Projects" list, but there is still the situation that @LucaDillenburg left/is leaving Dextra. wdyt?

**Comment by LucaDillenburg** [comment]
> @erickzanardo @gustavobigardi I intent to close this PR. We could pull it to the upcoming "Promising Projects" list, but there is still the situation that @LucaDillenburg left/is leaving Dextra. wdyt?

Hi Ohager,
I updated the README of the repositories. Now, it only has what has already being developed, no more "TODOs". Everything seems to be working fine. Can you take a look?
Thanks

**Comment by erickzanardo** [comment]
Not all requested changes were solved (the alphabetical order). Since this PR is a little old I am closing it.

**Pull request closed without merging**